src/components/CompareResults/beta/DownloadButton.tsx

```diff
@@ -13,10 +14,58 @@ const styles = {
   }),
 };
 
+interface GroupedResults {
+  [key: string]: object[];
+}
+
+interface ResultObject {
+  header_name: string;
+}
+
 function DownloadButton() {
+  const results = useAppSelector((state) => state.compareResults.data);
```

Comment:
This is good in case the activeComparison is 'All revisions' otherwise we should something like this
```suggestion
  const results = useAppSelector((state) => state.compareResults.data[activeComparison]);
```

Reply:
Good catch!